modules/stitching/src/camera.cpp

```diff
@@ -66,7 +66,7 @@ Mat CameraParams::K() const
     Mat_<double> k = Mat::eye(3, 3, CV_64F);
     k(0,0) = focal; k(0,2) = ppx;
     k(1,1) = focal * aspect; k(1,2) = ppy;
-    return std::move(k);
+    return k;
```

Comment:
Maybe so?
```suggestion
    return Mat(k);
```
Or the whole function should be rewritten, so that local variable being returned would match returned type.

Reply:
Yes, it works well, thank you !  GCC13 suggested to remove `std::move()` because it is redundant code. However clang on macOS/xcode  suggests to keep `std:move()` because type conversion. I think using `Mat()` is simple and the best instead of supressing warning.